xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
I had a look back through the git log for this file but I could only go back to 25/1/2011 and could not see any changes to this function.
I am not sure what or how to regression test this with no idea where to start with what to look for.